pkg/api/utils/resourceUtils.go

```diff
@@ -53,10 +55,81 @@ type resourceRequest struct {
 	Resources []*models.Resource `json:"resources"`
 }
 
-type GetOption func(url string) string
+// ResourceScope contains the necessary information to get a resource
+type ResourceScope struct {
+	project  string
+	stage    string
+	service  string
+	resource string
+}
+
+//NewResourceScope returns an empty ResourceScope to fill in calling Project Stage Service or Resource functions
```

Comment:
pls put a space between // and the beginning of the comment.
```suggestion
// NewResourceScope returns an empty ResourceScope to fill in calling Project Stage Service or Resource functions
```